src/Uno.UI.Tests/ControlTests/Given_Control.cs

```diff
@@ -13,46 +13,62 @@ namespace Uno.UI.Tests.ControlTests
 	public partial class Given_Control
 	{
 		[TestMethod]
-		public void When_ManualyApplyTemplate()
+		public void When_ImplicitlyApplyTemplate()
 		{
-			var templatedRoot = default(UIElement);
-			var sut = new MyControl
+			try
 			{
-				Template = new ControlTemplate(() => templatedRoot = new Grid())
-			};
-
-			Assert.IsNull(sut.TemplatedRoot);
-			Assert.IsNull(templatedRoot);
-
-			new Grid().Children.Add(sut); // This kind-of simulate that the control is in the visual tree.
+				FeatureConfiguration.Control.UseLegacyLazyApplyTemplate = true;
+				var templatedRoot = default(UIElement);
+				var sut = new MyControl
+				{
+					Template = new ControlTemplate(() => templatedRoot = new Grid())
+				};
 
-			Assert.IsNull(sut.TemplatedRoot);
-			Assert.IsNull(templatedRoot);
+				Assert.IsNull(sut.TemplatedRoot);
+				Assert.IsNull(templatedRoot);
 
-			var applied = sut.ApplyTemplate();
+				new Grid().Children.Add(sut); // This kind-of simulate that the control is in the visual tree.
 
-			Assert.IsTrue(applied);
-			Assert.IsNotNull(sut.TemplatedRoot);
-			Assert.AreSame(templatedRoot, sut.TemplatedRoot);
+				Assert.IsNull(sut.TemplatedRoot);
+				Assert.IsNull(templatedRoot);
+				
+				sut.Visibility = Visibility.Collapsed;
```

Comment:
We are testing a Uno side effect, at least we should add a comment, and keep valid tests as ignored

Reply:
The original tests were invalid regarding UWP behaviour (ApplyTemplate() will work without a parent), I'll try to restore them in a valid form.

Reply:
@dr1rrb How about now?